front/app/src/Router/VisitorRouter.tsx

```diff
@@ -11,6 +11,7 @@ import ReduxTestView from "../Views/ReduxTestView/ReduxTestView";
 import Home from "../Views/Home/Home";
 import Signin from "../Views/Signin/Signin";
 import BaseViewFromViteJs from "../Views/BaseViewFromVitejs/BaseViewFromViteJs";
+import TheGame from "../Views/TheGame/TheGame";
```

Comment:
Technically, the Game isn't accessible for Visitors (user not logged in)
see Switching Router at https://github.com/AdrianWeaver/transcendence/pull/34